pkg/scanner/options.go

```diff
@@ -1,7 +1,7 @@
 package scanner
 
 import (
-	"github.com/aquasecurity/defsec/pkg/scanners/options"
+	"github.com/aquasecurity/trivy-iac/pkg/scanners/options"
```

Comment:
Should we have trivy-iac as a dependency?

Reply:
hmm yeah maybe the `options` package needs to be abstracted out of `trivy-iac` as well to a common place. What do you think?

Reply:
I updated it here https://github.com/aquasecurity/trivy-aws/pull/8/commits/ba16ec1c71e6bd709773ee67047cd2d5ddc2d75b